x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -63,7 +63,11 @@ function stringMatch(str: string | undefined, substr: string) {
   );
 }
 
-export const DataFrameAnalyticsList: FC = () => {
+interface Props {
+  isManagementTable?: boolean;
```

Comment:
Naming nit: I'm afraid `isManagementTable` could be a bit confusing. It refers to the management section in Kibana, but seen alone here like this "management" could mean the opposite as in the capability to manage jobs where the table in Kibana management is view only. I'd like to suggest that we rename it to something that refers to the feature that the flag will enable/disable, something like `isReadOnly` or `hideActions`, what do you think?

Reply:
Good point. 🤔 I'm not sure `isReadOnly` or something like that would fit since it's more that we're showing a different level of detail as well as removing actions when it's for the Kibana management view. This is the same prop name used in for the job management table so it would be consistent. Happy to change to something that makes more sense. Maybe in a small follow-up once we've decided on a name?